modules/data_integrity_flag/help/data_integrity_flag.md

```diff
@@ -1,5 +1,14 @@
-# Data Integrity
+# Data Integrity Flag
 
-The Data Integrity module provides a direct way for users to view and update behavioural feedback at-a-glance, without requiring the user to navigate to the individual instrument forms or use the behavioural feedback pop-up window.
-The Selection Filter allows users to search for existing feedback for a particular Visit Label, Instrument and User. Upon clicking "Show Data", the user can add a new flag or behavioural feedback comment within the table directly below the Selection Filter. Users can also search through existing behavioural feedback results within the third table on the page. When adding new behavioural feedback, the date on which the flag was created must be indicated, as well as the Flag Status from the dropdown menu, and any additional comments. The "Save" button must be clicked in order to save the new behavioural flag. Click "Show updated data" or refresh the page to see the most recent behavioural flag within the results table.
-Within the results table (third table on the Data Integrity page), links under the "Instrument" column will redirect the user to a detailed summary of each field for that particular instrument along with links to behavioural feedback and to incomplete candidates. Users can also view the date the flag was submitted, the Flag Status, any comments, data cleaning feedback, and UserID. Note, Flag Status is based on the codes used in the dropdown of the second table, where 1=Ready for Review, 2= Review Completed, 3=Feedbacks Closed, and 4=Finalization.
+This module provides a direct way to view and update behavioural feedback at-a-glance.
+
+In the **Browse** tab, use the *Selection Filter* section to search for existing feedback. You can click any link in the *Instrument* column to view a detailed summary of each field for that instrument and visit.
+
+In the **Update** tab, you can add a new instrument Flag status. 
+You'll notice that  Flag Status is coded as: <br>
```

Comment:
Is `<br>` the only way to create new lines? I don't think we want HTML tags to be present in these files.

Reply:
I believe aside from adding a space between lines, the <br> tag is the only way to break up those lines with the dashes - and those would look silly with spaces between